static/src/javascripts/es6/projects/common/modules/crosswords/series.js

```diff
@@ -0,0 +1,28 @@
+/*eslint-disable no-new*/
+
+import qwery from 'qwery';
+import config from 'common/utils/config';
+import proximityLoader from 'common/utils/proximity-loader';
+import Series from 'common/modules/onward/onward-content';
+
+function insertOrProximity(selector, insert) {
+    if (window.location.href) {
+        insert();
+    } else {
+        const el = qwery(selector)[0];
+
+        if (el) {
+            proximityLoader(el, 1500, insert);
+        }
+    }
+}
+
+
+export default() => {
+
+    insertOrProximity('.js-onwards', function () {
```

Comment:
this looks like a copy paste, i don't think you need insertOrProximity at all here. Pop all the code into default.